src/app/rundown-overview/components/rundown-overview/rundown-overview.component.scss

```diff
@@ -1,3 +1,14 @@
+:host {
+  height: 100vh;
```

Comment:
I would appreciate if we did not make assumptions based on the window size, but rather the parent element, so this would become `100%`.

Reply:
Fixed